packages/server/graphql/public/mutations/loginWithPassword.ts

```diff
@@ -0,0 +1,37 @@
+import {AuthenticationError} from 'parabol-client/types/constEnums'
+import encodeAuthToken from '../../../utils/encodeAuthToken'
+import {MutationResolvers} from '../resolverTypes'
+import attemptLogin from '../../mutations/helpers/attemptLogin'
+
+const loginWithPassword: MutationResolvers['loginWithPassword'] = async (
+  _source,
+  {email, password},
+  context
+) => {
+  if (process.env.AUTH_INTERNAL_DISABLED === 'true') {
```

Comment:
-1 You did a nice job moving this to `permissions.ts` for `signUpWithPassword`, any reason this should stay here for this mutation?

Reply:
I overlooked this file! Moved over.